README.md

```diff
@@ -220,6 +220,16 @@ composer train/train.py \
   eval_interval=0 \
   save_folder=mpt-125m
 
+# To iterate on training on CPU run command (dev only):
```

Comment:
Can we not add this to the readme? Would prefer to keep on simple path through the quickstart?

Reply:
removed.